com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/lightlistbuild.compute

```diff
@@ -26,6 +26,7 @@
 
 #define FINE_PRUNING_ENABLED
 #define PERFORM_SPHERICAL_INTERSECTION_TESTS
+#define USE_LIGHTVOLUME_CACHE
```

Comment:
Should I disable this for switch? since I have only tested on msft/sony consoles.

Reply:
We don't support switch

Reply:
I would delete the old code path, or if we want to keep it for other reasons (Don't see any if you tested on consoles and it is always faster :) ) we should factor out the processing of the light given that it looks the same as before, just the outer part of the loop is different and at any change we'd have to maintain two copies of the same code. 

But then again, I vote for deleting old path instead :D